lib/authlogic/acts_as_authentic/login.rb

```diff
@@ -178,13 +178,13 @@ def find_with_case(field, value, sensitive)
             elsif ar_gem_version >= Gem::Version.new('5.0')
               connection.case_sensitive_comparison(arel_table, field.to_s, columns_hash[field.to_s], value)
             else
-              if ar_gem_version < Gem::Version.new('4.2')
-                value = connection.case_sensitive_modifier(value)
+              value = if ar_gem_version < Gem::Version.new('4.2')
+                connection.case_sensitive_modifier(value)
               else
-                value = connection.case_sensitive_modifier(value, field.to_s)
-              end
+                connection.case_sensitive_modifier(value, field.to_s)
+                      end
               arel_table[field.to_s].eq(value)
-            end
+                       end
```

Comment:
The indentation here seems strange to me. I would prefer something like:

```
value = if ar_gem_version < Gem::Version.new('4.2')
          connection.case_sensitive_modifier(value)
        else
          connection.case_sensitive_modifier(value, field.to_s)
        end
```

Or, you can extract a method:

```
def _connection_case_sensitive_modifier(value, field)
  if ar_gem_version < Gem::Version.new('4.2')
    connection.case_sensitive_modifier(value)
  else
    connection.case_sensitive_modifier(value, field.to_s)
  end
end

value = _connection_case_sensitive_modifier(value, field)
```